### ISSUE TITLE
Duo GPU Capabilities?

### ISSUE THREAD

**Issue by BlrFox** [karpathy/char-rnn#232]
My Computer runs on 2 GPUs 'Intel UHD 630' and 'AMD Radeon Pro 560X'

I noticed when training using the GPU option only one of my gpu was peaking while the other one runs quietly. Would you thing there eventually be an equal distribution between the multiple parts?

--Thanks :3

**Comment by greydoubt** [comment]
No, they're different chipsets, the UHD is a built-in one and very slow for the tasks so them "evening out" could mean the average performance of the two, or, the speed of the slowest one.

**Comment by BlrFox** [comment]
Yea you're probably right, don't know if this makes a difference or not, but they came both built in.

**Comment by greydoubt** [comment]
It'll be there for when your main GPU bites the dust I'm sure!

**Comment by BlrFox** [comment]
Lol I was assuming that tbh x3, would using cpu be better for a longer run? 'Intel i7, 12 core'
